spi/identity-hub-spi/src/main/java/org/eclipse/edc/identityhub/spi/credentials/model/PresentationEnvelope.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright (c) 2023 GAIA-X
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       GAIA-X - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.identityhub.spi.credentials.model;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.edc.spi.result.Result;
+
+/**
+ * The {@link PresentationEnvelope} it's used to wrap an implementation of verifiable presentation with a given format.
+ */
+public interface PresentationEnvelope {
```

Comment:
See comment above: we should have a unified envelope for both VP and VC

Reply:
Using same envelope now for both vp and vc with a list of credentials